aws-lc-rs/src/aead.rs

```diff
@@ -678,7 +1041,7 @@ impl Algorithm {
     /// The length of the nonces.
     #[inline]
     #[must_use]
-    pub fn nonce_len(&self) -> usize {
+    pub const fn nonce_len(&self) -> usize {
```

Comment:
By making this a `const` function, we're prevented from ever having it return something other than a `const` value (unless we break backwards compatibility). This could prevent us from supporting certain algorithms through this API in the future.